setup.sh

```diff
@@ -14,6 +14,10 @@ normalize_version() {
 }
 
 latest_version() {
+  jq --arg arch "$ARCH" '.releases | map(select(.dart_sdk_arch == null or .dart_sdk_arch == $arch)) | first'
+}
+
+latest_channel_version() {
```

Comment:
Would I rename `latest_channel_version()` as `channel_latest_version()`?

Reply:
I think it's ok